app/controllers/user.js

```diff
@@ -7,6 +7,7 @@ const bcrypt = require('bcryptjs'),
   errors = require('../errors');
 
 const saltRounds = 10;
+let tokenSerial = 1;
```

Comment:
You are never using this variable outside of `expireAllUsers` so it doesn't makes any use to have it here.